tests/cli/test_subcommands.py

```diff
@@ -210,11 +214,11 @@ def test_env_remove(conda_cli: CondaCLIFixture, tmp_env: TmpEnvFixture):
         assert not code
 
 
+@pytest.mark.benchmark
```

Comment:
In the future I'd rather see a `tests/benchmarks/` directory with benchmark-focused tests so we don't mix IO too much, since codspeed since to be focusing on CPU bound tasks.

Reply:
Yeah, that makes sense, I even started with it at first and then noticed that I was duplicating a lot of efforts. I started from the tests in conda-benchmarks, but thought to first get some experience with codspeed.